examples/README.md

```diff
@@ -25,6 +25,8 @@ before trying out the examples.
 
 - [Json serialization](src/main/java/io/grpc/examples/advanced)
 
+- [Google Authentication](GOOGLE_AUTH_EXAMPLE.md)
```

Comment:
As this pulls in pub-sub and Google Cloud account as dependency, it might be better to move it from Basic Examples section to Other Examples section, and move the code to a separate project, in parallel with tls, alts, android, kotlin.

Reply:
Yes, good point. Let me look into it

Reply:
moved